source/backend/entities/Partials/StandardIdentityBaseAppEntity.cs

```diff
@@ -0,0 +1,10 @@
+namespace Pims.Dal.Entities
+{
+    /// <summary>
+    /// StandardIdentityBaseAppEntity abstract class, provides an internal representation of the class id as an standard member.
+    /// </summary>
+    public abstract class StandardIdentityBaseAppEntity<T>
+    {
+        public abstract T Internal_Id { get; set; }
+    }
```

Comment:
This should make it more clear the purpose of this abstract class. I.e. to standardize the access to the Id property.

Reply:
@asanchezr I've removed the inheritance to IIdentityEntity given they fulfill different needs in the code. Let me know if you think this is a problem.

Reply:
As long as existing code continues to work as-is I'm okay